pkg/util/system/file_linux.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package system
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+func CountOpenFileDescriptors(procPath string, pids []int) (uint64, bool) {
```

Comment:
This needs a docstring.  Also, this seems quite similar to `GetFileDescriptorLen`, yet its name is of a different shape.  Could those be changed so that the similarity is clear?  `CountProcessFileDescriptors` and `CountProcessesFileDescriptors` perhaps?

Reply:
These functions also need some tests -- what if the pid doesn't exist, etc.

Reply:
Everything should good now